Yoast/wpseo-woocommerce
# Add product:condition to our OG output

## Summary

This PR can be summarized in the following changelog entry:

* Add `product:condition` to the OpenGraph output. It defaults to `new` but can be filtered using the new `Yoast\WP\Woocommerce\product_condition` filter.

## Relevant technical choices:

* Introduced a new `Yoast\WP\Woocommerce\product_condition` filter.

## Test instructions

This PR can be tested by following these steps:

* Go to a product page, see the product now has `product:condition` in its output.

Fixes #313
